### ISSUE TITLE
Account for bolus velocity in MOC AM when GM is turned on

### ISSUE THREAD

**Issue by milenaveneziani** [MPAS-Dev/MPAS-Model#55]
The MOC online analysis member currently does not take into account the bolus velocity component when computing the streamfunction, in cases when the GM parameterization is on. This should be added to the AM, in order to have the total MOC result.

**Comment by xylar** [comment]
@maltrud, I think you had said at a meeting while I was still in Los Alamos that you would be willing to help with this development, correct?  No pressure, just making sure I remember correctly.

**Comment by maltrud** [comment]
@xylar , yes you are right.  i offered to help, but of course forgot about it myself.  thanks for the reminder.
